.github/workflows/sdk-publish.yaml

```diff
@@ -268,3 +279,41 @@ jobs:
           footer: "Linked Repo <{repo_url}|{repo}> | <{run_url}|View Run>"
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  publish-gems:
+    if: ${{ needs.release.outputs.ruby_regenerated == 'true' && inputs.publish_ruby == 'true' }}
+    name: Publish Ruby SDK
+    runs-on: ubuntu-latest
+    needs: release
+    defaults:
+      run:
+        working-directory: ${{ needs.generate.outputs.ruby_directory }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@ec02537da5712d66d4d50a0f33b7eb52773b5ed1
+        with:
+          ruby-version: "3.1"
+        working-directory: ${{ needs.release.outputs.ruby_directory }}
```

Comment:
this line `working-directory: ${{ needs.release.outputs.ruby_directory }}` can be omitted as you are using the defaults above